packages/grafana-ui/src/components/Table/FilterList.tsx

```diff
@@ -169,19 +169,20 @@ export const FilterList = ({
   }, [onChange, values, items, selectedItems]);
 
   return (
-    <VerticalGroup spacing="md">
+    // spacing="md"
```

Comment:
Do we want / need this commented out code anymore?

Reply:
Nope, good catch!